docker/load_data.sh

```diff
@@ -8,6 +8,7 @@ cd /opt/geotrek-admin
 ./manage.py loaddata cirkwi
 ./manage.py loaddata basic
 ./manage.py loaddata licenses
+./manage.py loaddata rules
```

Comment:
Same as postinst for Debian. A user that will install from scratch with docker it's Geotrek will have an error launching this script because by default the sensitivity module is not activated. I would remove this line.